opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -99,6 +100,46 @@ def get_context(self) -> 'SpanContext':
         """
 
 
+class TraceOptions(int):
+    """A bitmask that represents options specific to the trace.
+
+    The only supported option is the "recorded" flag (``0x01``). If set, this
+    flag indicates that the trace may have been recorded upstream.
+
+    See the `W3C Trace Context`_ spec for details.
+
+    .. _W3C Trace Context: https://www.w3.org/TR/trace-context/#trace-flags
+    """
+    DEFAULT = 0x00
```

Comment:
We are using `SAMPLED` and `UNSAMPLED` names in [node library](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-types/src/trace/trace_options.ts#L24), not sure If we want to be consistent across.

Reply:
Making these consistent sounds like a good idea. FWIW java's using `DEFAULT_OPTIONS` and `IS_SAMPLED` and the [W3C spec](https://www.w3.org/TR/trace-context/#recorded-flag-00000001) calls this "recorded". Do you know why we don't just use the W3C language?

Reply:
+1 on "just use the W3C language".
This is subjective and in most cases people just implement something based on personal taste/bias. Now we have a W3C standard, it'll be good to align with it.

Reply:
After looking at the W3C spec again I think this should be called "recorded", but then we should also be calling span IDs "parent IDs".

Reply:
In the HTTP header, the ID of the span that should be the parent of any span that is created when handling this request is transported. That's why it is called parent.